community/CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# Contributing to SourceCred
+
+Contributing to SourceCred means any of the following:
+
+ - opening an issue to ask a question, or request a feature
+ - open a pull request (PR) to request for review
+
+See the [glossary](glossary.md) or [introduction to github guide](../guides/GITHUB.md) 
+if you have questions about terms used in this document.
+
+## Scope of Contribution
+
+This contribution guide covers the repositories (projects) under the 
+[SourceCred GitHub organization](https://github.com/sourcecred). While this contributing guide
+is generally applicable, each repository can potentially be different, and it's
+recommended to check within the repository you are interested
+in contributing to for a more detailed guide to contributing.
+
+## Opening Issues
+
+You can open an issue on any SourceCred project to ask a question, suggest a feature,
+or report a bug. If you want to have discussion before opening the issue, you are welcome 
+and encouraged to chat with others on [discourse](discourse.sourcecred.io/h), [discord](https://discordapp.com/channels/453243919774253079/670490733077332000), or any other method that is available and comfortable for you
```

Comment:
Discourse link is broken; should be https://discourse.sourcecred.io
Discord link doesn't look like an invite link; should be https://sourcecred.io/discord-invite

nit: would prefer that the urls not be inline, i.e. just define  `[discourse]: https://discourse.sourcecred.io` on its own line.

Reply:
I copied both from my browser, which was clearly wrong to do!